tests/sentry/relay/test_config.py

```diff
@@ -462,7 +459,10 @@ def test_project_config_with_boosted_latest_releases_boost_in_dynamic_sampling_r
         )
         release_ids.append(release.id)
 
-    boosted_releases = [[release_ids[0], ts - BOOSTED_RELEASE_TIMEOUT * 2]]
+    # We mark the first release (1.0) as expired.
+    boosted_releases = [
+        [release_ids[0], ts - Platform(default_project.platform).time_to_adoption * 2]
```

Comment:
just for readability you can use new variable:
```
time_to_adoption = Platform(default_project.platform).time_to_adoption
```

`boosted_releases = [[release_ids[0], ts - time_to_adoption * 2]]`